app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/MainActivity.kt

```diff
@@ -273,10 +273,8 @@ class MainActivity : BaseActivity(),
             MyApp.task_running = TASKS.FETCH
             showFetchingStatus()
             val url = appRepository.readScheduleUrl()
-            val okHttpClient = CustomHttpClient.createHttpClient()
```

Comment:
:broom: Remove unused `import nerd.tuxmobil.fahrplan.congress.net.CustomHttpClient`

Reply:
:heavy_check_mark: Removed in 081c1fa9cd6d96f7a89ec4ab155e6d1700c04f25.